keras_cv/models/segmentation/deeplab.py

```diff
@@ -67,12 +73,21 @@ def __init__(
         backbone_weights=None,
         spatial_pyramid_pooling=None,
         segmentation_head=None,
+        segmentation_head_activation="softmax",
+        input_shape=(None, None, 3),
+        input_tensor=None,
         **kwargs,
     ):
-        super().__init__(**kwargs)
 
-        self.classes = classes
-        # ================== Backbone and weights. ==================
+        inputs = utils.parse_model_inputs(input_shape, input_tensor)
+        height = input_shape[0]
+        width = input_shape[1]
+
+        x = inputs
+
+        if include_rescaling:
+            x = layers.Rescaling(1 / 255.0)(x)
```

Comment:
just a nit that we're moving away from `backbone` as a string and always accepting `backbone` as a `keras.Model`, so we no longer need `backbone_weights` and `include_rescaling` here

Reply:
Makes sense, updating